docs/README.md

```diff
@@ -30,8 +30,11 @@
 | | Command | Notes |
 | - | - | - |
 | PyPI | `pipx install sherlock-project` | `pip` may be used in place of `pipx` |
-| Homebrew | `brew install sherlock` | Community supported |
 | Docker | `docker pull sherlock/sherlock` | |
+| Debian family | `apt install sherlock` | Kali, Parrot, BlackArch, Testing, Sid |
```

Comment:
BlackArch is apart of Arch Linux and not Debian

> BlackArch Linux is an **Arch Linux-based** penetration testing distribution for penetration testers and security researchers. 

Source: https://blackarch.org/

Reply:
Agh. Working on autopilot. Will adjust.

Reply:
Removed BlackArch